src/commands/transaction/pullFunds.ts

```diff
@@ -54,7 +54,7 @@ export class CommandOptions extends ProfileOptions {
     message: string;
 
     @option({
-        flag: 'x',
+        flag: 'M',
```

Comment:
not sure we should use lowercase or not. as seen lower cases were used in other PRs